### PR TITLE
[SOL-200] Transfer to reference account flow

### PR DESCRIPTION
**Pull request by ilie-lupu-ts**
## Description

Transfer to reference account flow. If the reference account is not set for the account, an error will be shown

## Video

https://github.com/ivoryio/SolarisDemoApp/assets/127083262/e45fa2e5-b105-435d-a4a4-a7c1883273cf

### CODE REVIEW

File: lib/redux/person/reference_account/reference_account_state.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+abstract class ReferenceAccountState extends Equatable {
+  @override
+  List<Object?> get props => [];
+}
+
+class ReferenceAccountInitialState extends ReferenceAccountState {}
+
+class ReferenceAccountLoadingState extends ReferenceAccountState {
+  ReferenceAccountLoadingState();
```
Comment (Tudor90) [inline lib/redux/person/reference_account/reference_account_state.dart]: Puteai sa il lasi ca "ReferenceAccountInitialState" adica fara constructor

Comment (ilie-lupu-ts) [reply lib/redux/person/reference_account/reference_account_state.dart]: Solved

**Review by szaszgyozo — APPROVED** [review]

File: lib/widgets/ivory_generic_error.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  final String errorMessage;
+  final TextStyle textStyle;
+
+  const IvoryGenericError({
+    super.key,
+    this.errorMessage = 'Something went wrong',
+    this.textStyle = const TextStyle(
+      color: Colors.red,
+      fontSize: 16,
+      fontWeight: FontWeight.bold,
```
Comment (Tudor90) [inline lib/widgets/ivory_generic_error.dart]: 1. Foloseste "w-urile" din design gen w600 in loc de bold, si poti sa pui in loc de asta textstyle ala declarat deja in configs

**Review by Tudor90 — APPROVED** [review]

File: lib/widgets/ivory_generic_error.dart (hunk 2 above)
Comment (ilie-lupu-ts) [reply lib/widgets/ivory_generic_error.dart]: Solved

**Pull request merged**